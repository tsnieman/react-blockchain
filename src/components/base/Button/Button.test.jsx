import React from 'react';
import renderer from 'react-test-renderer'; // eslint-disable-line import/no-extraneous-dependencies
import Button from './Button';

test('Button is tested', () => {
  const component = renderer.create(
    <Button />,
  );

  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
