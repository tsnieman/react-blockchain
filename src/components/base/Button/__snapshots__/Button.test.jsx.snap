exports[`test Button is tested 1`] = `
<button
  className=""
  disabled={false}
  styleName="button-default" />
`;
